packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -55,40 +66,41 @@ export const Select = styled( Ariakit.Select, {
 		const sizes = {
 			compact: {
 				[ heightProperty ]: 32,
-				paddingInlineStart: space( 2 ),
-				paddingInlineEnd: space( 1 ),
+				paddingInlineStart: 8,
```

Comment:
Is there a good reason to stop using the grid space util and hardcode the padding widths like that?

Reply:
Curious about that, too.

Reply:
No big reason other than that I thought it was simpler to read in context, especially when I want to add the `chevronIconSize` like this. (`space()` returns a `calc()`).